### DISCUSSION TITLE
Laragon - WordPress - Hmmm… can't reach this page

### DISCUSSION THREAD

**Discussion by tienloc1** [leokhoa/laragon#706] [General]
I'm not sure what's wrong when I'm working with those localhost sites and all the websites are not accessible now. Maybe I've mistake and updated something so it's causing the problem. I'm not sure how to debug it at this moment and I hope to get some help here, thank you to everyone.

======== Update

If I tried to disable the SSL then the website accessible. The SSL is enabled in Apache with Add laragon.crt to Trust Store, the SSL setting is enabled with port 443, and the PHP extension OpenSSL is enabled, I'm not sure what's going on cause the SSL URL will not work.

**Comment by KingWebsites** [comment]
Have you tried reloading Apache?

**Reply by tienloc1** [reply]
I've tried clicking on the Reload many times, but it's not working. I'm temporarily disabling the SSL to work with my WordPress site now.

**Comment by IsAmirMmd** [comment]
do you have multiple wordpress website on your /www folder?

**Reply by tienloc1** [reply]
Yes, I have around 20+ websites, I'm working as a supporter so I need to install the new WP site often.

**Comment by phu-huy** [comment]
I am working with laragon 6 installation version, using with windows 11 computer version 23H2
When I installed wordpress with default option an error occurred!

1. I tried starting windows, creating a project and reloading the server many times but it didn't work.
2. I am trying a php version 7.4.19. Still not working
3. I tried running nginx server according to the default version. Still not working

At any time, there is a solution that will help me and those who come after me. Laragon has not had any updates or patches in the last 2 years!
![image](https://github.com/leokhoa/laragon/assets/95569554/9b33b406-49b5-4923-8e55-fe80083b30c6)
![image](https://github.com/leokhoa/laragon/assets/95569554/899f0878-3973-499d-b113-959058772b12)
![image](https://github.com/leokhoa/laragon/assets/95569554/2b3d7e6b-77e0-41db-b36b-5ca7063fc08d)

**Reply by tienloc1** [reply]
In my case, it's working if I'm disabling the SSL mode.

**Comment by IsAmirMmd** [comment]
as i know it's a common bug between users and I haven't found any solutions yet

**Reply by tienloc1** [reply]
I have been using Laragon for a few years without any issues, but this is the first time I have encountered this problem and I cannot find a solution. I hope @leokhoa can look into it when he has some free time.

**Comment by wolfiejunk** [comment]
Are the sites Wordpress sites? And, are you trying to access them from computers on a LAN other than the one that is hosting the sites?

**Reply by tienloc1** [reply]
Hmm, apologies for my confusion here, I'm not certain what you mean by 'accessing the site on a LAN.' All sites are powered by WordPress.

**Reply by wolfiejunk** [reply]
Hi,

What I meant was that I could access the WP sites on the computer with Laragon installed, but not on others connected on the LAN. 

I am not sure why that was. It worked fine, prior to a Win11 upgrade and then the issue began. After many attempts to solve it by changing settings, I finally deleted Laragon and reinstalled it. That took care of the problem.

Rich

**Comment by ShaungBhone** [comment]
I got the error with all app. Pretty url is not working in my window 11 msi.

**Reply by ghost** [reply]
It may be occurring because the virtual hosts are not configured correctly or the WordPress URL is not configured correctly in the WordPress settings. Review the error logs for more information about your problem and also check for port conflicts. Laragon usually handles port conflicts automatically, but it's good to check if there are any conflicts causing issues. Additionally, test with a simple PHP file. I hope this information helps you.
